### ISSUE TITLE
Definition of joint contracture

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7688]
This is the proposed definition of the 'ancestor' term of all contracture terms, Joint Contracture (which will be  superior to [Flexion contracture HP:0001371](https://hpo.jax.org/app/browse/term/HP:0001371)).

Definition: A joint contracture is a limitation in the passive range of motion of the joint due to shortening of the periarticular connective tissues and muscles. A contracture prevents movement of the associated body part.
PMID:[ 22938881](https://pubmed.ncbi.nlm.nih.gov/22938881)
PMID:[ 30928194](https://pubmed.ncbi.nlm.nih.gov/30928194)
PMID: 26247029

Comment: To assess a joint contracture, the passive or active range of motion (ROM) of a joint with a contracture is assessed by moving the joint through its natural range with no active effort from the individual. The angular distance of the joint motion and the loss of ROM in a contracture is recorded through comparison with the contralateral joint or normative values. Conventionally, a joint contracture is named according to the joint involved and the direction opposite the lack of range. For example, inability to fully extend the knee is referred to as a knee flexion contracture, compared to inability to bend (flex) the knee, which is referred to as a knee extension contracture. Contractures in neuromuscular diseases develop due to intrinsic myotendinous structural changes and extrinsic factors. Known contributing extrinsic factors include decreased ability to actively move a limb through its full range of motion, immobility or static positioning for prolonged periods of time, and agonist antagonist muscle imbalance. Intrinsic muscle tissue alterations in dystrophic myopathies contribute to contracture formation. The most significant histologic changes are those of muscle fiber loss, abnormal residual dystrophic muscle fibers, segmental necrosis of muscle fibers, and increased amounts of adipose tissue, connective tissue, and fibrosis.

**Comment by pnrobinson** [comment]
added
